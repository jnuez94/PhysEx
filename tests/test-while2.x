int i;
int j;
int k;
void func main() {
	i = 0;
	k = 0;
	while(i < 3) {
		for (j = 0; j < 3; j = j + 1) {
			while (k < 3){
				print("slam");
				k = k + 1;
			}
			print("dank");
			k = 0;
		}
		print ("memes");
		i=i+1;
	}
}
