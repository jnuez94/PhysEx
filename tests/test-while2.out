slam
slam
slam
dank
slam
slam
slam
dank
slam
slam
slam
dank
memes
slam
slam
slam
dank
slam
slam
slam
dank
slam
slam
slam
dank
memes
slam
slam
slam
dank
slam
slam
slam
dank
slam
slam
slam
dank
memes
